src/app/SharpRaven/RavenClient.cs

```diff
@@ -395,7 +406,12 @@ private string HandleException(Exception exception, Requester requester)
 
                 var response = webException.Response;
                 id = response.Headers["X-Sentry-ID"];
-                if (string.IsNullOrWhiteSpace(id))
+#if net35
+                var isNullOrWhiteSpace = Utilities.SystemUtil.IsNullOrWhiteSpace(id);
+#else
+                var isNullOrWhiteSpace = String.IsNullOrWhiteSpace(id);
+#endif
```

Comment:
Yes, but I thought you would prefer the original official .net implementation if it was available.